README.md

```diff
@@ -29,6 +29,7 @@ Search for "Open Policy Agent" in the Extensions (Shift ⌘ X) panel and then in
 | --- | --- | --- |
 | `opa.path` | `null` | Set path of OPA executable. |
 | `opa.checkOnSave` | `false` | Enable automatic checking of .rego files on save. |
+| `opa.strictMode` | `false` | Enable [strict-mode](https://www.openpolicyagent.org/docs/latest/strict/) for the `OPA: Check File Syntax command`. |
```

Comment:
What do you think about defaulting to true? It would be nice to turn these on in as many places as possible to slowly drive down the number of instances where users rely on constructs that we want to remove eventually.

If we had some way to notify people that a particular error was caused due to strict mode then we could notify let them know how to turn off strict mode when those errors occur (but only as a last resort).

Reply:
I can get behind that philosophy 👍 .

Perhaps we could introduce a new error type for this in OPA, like `rego_strict_compile_error`. That way we, and other consumers of the command-line - or even SDK, could respond accordingly.

Reply:
That would need to wait until the next OPA release though, If that's what we want. Meanwhile/alternatively, we could just check for the specific error phrases in the output. That's not as robust, though, and will break when OPA 1.0 is released and these errors are no longer part of strict-mode, or if the error messages are changed before then.